Seneca-CDOT/starchart
# Initialize Chakra Theme

### What I did?
I created the folder `theme` where we gonna have all theme-related files. For now, it has `index.ts` which imports all other files and initializes the theme itself. Another file is `colors.ts` where I added the Seneca colours in spectra from 100 to 900 and that colour I called `brand` (this is how chakra does it in the example in the doc so probably this is the way to go in terms of naming). I also created the button on `index.ts` just to show how the component that uses the theme looks like 

### How to test?
Just run the app locally and check the front end, you should see the button with very familiar color
